components/oauth/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/token/OAuth2TokenEndpoint.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2013, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
  *
```

Comment:
I think the licence year needs to updated everywhere, in some components it's 2010
like here : 
https://github.com/wso2/carbon-identity/blob/master/components/claim-mgt/org.wso2.carbon.claim.mgt/src/main/java/org/wso2/carbon/claim/mgt/dto/ClaimAttributeDTO.java